Server/Components/NetworkClient.hpp

```diff
@@ -0,0 +1,20 @@
+/*
+** EPITECH PROJECT, 2022
+** R-Type
+** File description:
+** NetworkClient
+*/
+
+#pragma once
+
+#include "Component/Component.hpp"
+
+namespace ecs
+{
+    class NetworkClient : public Component {
+      public:
+        std::size_t id;
```

Comment:
In convention, variable must be put under variable declaration

Btw, some empty line could be usefull for reading

Reply:
It's ok in a component.